msticpy/vis/entity_graph_tools.py

```diff
@@ -33,6 +35,8 @@
 req_alert_cols = ["DisplayName", "Severity", "AlertType"]
 req_inc_cols = ["id", "name", "properties.severity"]
 
+_BOKEH_VERSION = tuple(int(i) for i in bokeh.__version__.split("."))
```

Comment:
I would implement this with:
```python
from importlib.metadata import version
from packaging.version import Version, parse

_BOKEH_VERSION: Version = parse(version("bokeh"))
```

Reply:
As always, I can rely on you for a more elegant solution 💯